katcp/test/test_server.py

```diff
@@ -275,8 +285,16 @@ class DeviceTestServerWrong(DeviceTestServer):
         with self.assertRaises(ValueError):
             DeviceTestServerWrong('', 0)
 
+katcp_version = __version__
 
 class test_DeviceServer(unittest.TestCase, TestUtilMixin):
+
+    expected_connect_messages = (
+            r'#version-connect katcp-protocol 5.0-IM',
+            # Will have to be updated for every library version bump
+            r'#version-connect katcp-library katcp-python-%s' % katcp_version,
```

Comment:
Does this comment still apply, if you're pulling the version from `__version__`, or are you referring to the `5.0-IM` line?